src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/FillBadData.cpp

```diff
@@ -0,0 +1,409 @@
+#include "FillBadData.hpp"
```

Comment:
You can implement the functor now with the `ExecuteDataFunction` function in `"complex/Utilities/FilterUtilities.hpp"` unless you were gonna parallelize this equation. in which case this should be done. See comment in Slack `complex` channel for a possible solution to this choice.